### ISSUE TITLE
node args are included in ng lint arguments 

### ISSUE THREAD

**Issue by tl24** [elunic/node-ng-lint-staged#5]
In the latest version 12, node and ng-lint-staged args are passed to "ng lint" task and causes errors.

package.json:
```
{
    "scripts": {
         "lint": "ng lint my-app --fix",
    },
    "lint-staged": {
    "*.ts": [
      "ng-lint-staged lint --"
    ]
  }
```
I receive an error when trying to run it:
```
✖ ng-lint-staged lint --:
Unknown option: '/home/myapp/node_modules/.bin/ng-lint-staged'

Unknown option: 'lint'
Unknown option: '/home/linuxbrew/.linuxbrew/Cellar/node@14/14.17.0/bin/node'

npm
 ERR! code ELIFECYCLE
npm ERR! 
errno 1

npm 
ERR! my-app@0.0.0 lint: `ng lint my-app --fix "/home/linuxbrew/.linuxbrew/Cellar/node@14/14.17.0/bin/node" "/home/myapp/node_modules/.bin/ng-lint-staged" "lint" "--lint-file-patterns=projects/my-app/src/app/core/some.service.ts" "--lint-file-patterns=projects/my-app/src/app/features/some.component.spec.ts"`
```

**Comment by tl24** [comment]
I noticed the arg loop switched in one of the commits:

Old:
`for (let i = 3, iz = process.argv.length; i < iz; i++) {`

New:
`for (const arg of process.argv) {
`

**Comment by whefter** [comment]
Whoops. Thanks for the hint. Reverted to old loop. Should be published in a couple of minutes (12.0.1 / 7.0.1)
